great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -85,6 +89,57 @@ class DataAssistant(metaclass=MetaDataAssistant):
         ...
     """
 
+    COMMONLY_USED_PARAMETER_BUILDERS: Dict[str, ParameterBuilder] = {
+        "table_row_count_metric_multi_batch_parameter_builder": MetricMultiBatchParameterBuilder(
```

Comment:
Same note as a prior PR about magic strings - could we avoid hardcoding this? Or perhaps uses constants or an enum? Between this dict and the self-initializing expectation configs, this becomes hard to maintain.

I think we could encapsulate these in a class and access through dot notation?

Reply:
@cdkini This has been addressed -- please take a look.  Thanks!